products/vuetify.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vuetify
+category: framework
+iconSlug: vuetify
+permalink: /vuetify
+alternate_urls:
+-   /vuetify
+
+versionCommand: npm list vuetify
+releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released
```

Comment:
The LTS link actually gives a better overview of all the supported releases, so using that makes sense perhaps?

```suggestion
releasePolicyLink: https://vuetifyjs.com/en/introduction/long-term-support/
```